src/App.js

```diff
@@ -0,0 +1,12 @@
+import './App.css';
+import Greeting from './components/Greeting';
+
+function App() {
+  return (
+    <div className="App">
+      <Greeting />
+    </div>
```

Comment:
- [ ] the App component should have a route that leads to the Greeting component.